README.md

```diff
@@ -4,15 +4,16 @@
 
 # Quokka
 
-<img src="docs/assets/quokka.png" alt="A happy quokka with style" width="300"/>  
```

Comment:
Lots of auto-formatting from prettier in here. I think worth keeping the changes.